src/Fixer/FunctionNotation/FunctionDeclarationFixer.php

```diff
@@ -24,11 +26,53 @@
  *
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
-final class FunctionDeclarationFixer extends AbstractFixer
+final class FunctionDeclarationFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
+    const SPACING_NONE = 'none';
```

Comment:
thanks so much, this PR is getting closer to getting merged :)
I would like to do some testing, could you please rebase on top of the master branch?